nncf/config/schemata/experimental_schema.py

```diff
@@ -115,9 +117,13 @@
         "filter_importance": with_attributes(
             STRING,
             description="The type of filter importance metric. Can be"
-            " one of `L1`, `L2`, `geometric_median`."
+            " one of `L1`, `L2`, `geometric_median`, `custom`."
```

Comment:
Maybe `manual` is a better name.

Reply:
@ljaljushkin I don't think "manual" captures the intent here. It could be an "custom" automated algorithm that generates the importance information. I like "external". What are your thoughts about "external"? It could be manual or automated :) Thanks!